joint_limits:
  arm_lift_joint:
    has_position_limits: true
    min_position: 0.0
    max_position: 0.69
    has_velocity_limits: true
    max_velocity: 0.1
    has_acceleration_limits: true
    max_acceleration: 0.1
  arm_flex_joint:
    has_position_limits: true
    min_position: -2.62
    max_position: 0.0
    has_velocity_limits: true
    max_velocity: 1.0
    has_acceleration_limits: true
    max_acceleration: 1.0
  arm_roll_joint:
    has_position_limits: true
    min_position: -2.09
    max_position: 3.84
    has_velocity_limits: true
    max_velocity: 1.0
    has_acceleration_limits: true
    max_acceleration: 1.0
  wrist_flex_joint:
    has_position_limits: true
    min_position: -1.92
    max_position: 1.22
    has_velocity_limits: true
    max_velocity: 1.0
    has_acceleration_limits: true
    max_acceleration: 1.0
  wrist_roll_joint:
    has_position_limits: true
    min_position: -1.92
    max_position: 3.67
    has_velocity_limits: true
    max_velocity: 1.0
    has_acceleration_limits: true
    max_acceleration: 1.0
  head_pan_joint:
    has_position_limits: true
    min_position: -3.84
    max_position: 1.75
    has_velocity_limits: true
    max_velocity: 1.0
    has_acceleration_limits: true
    max_acceleration: 1.0
  head_tilt_joint:
    has_position_limits: true
    min_position: -1.57
    max_position: 0.52
    has_velocity_limits: true
    max_velocity: 1.0
    has_acceleration_limits: true
    max_acceleration: 1.0
  hand_motor_joint:
    has_position_limits: true
    min_position: -0.8
    max_position: 1.24
    has_velocity_limits: true
    max_velocity: 1.0
    has_acceleration_limits: true
    max_acceleration: 1.0
